src/store/store.js

```diff
@@ -0,0 +1,11 @@
+import { createStore } from 'redux';
+import rootReducer from '../reducers/index';
+
+const configureStore = () =>
+  createStore(
+    rootReducer,
+    // eslint-disable-next-line no-underscore-dangle
```

Comment:
Let's not disable this rule here. We have an `.eslint.js` file where it is more appropriate to do this